babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    presets: ["@babel/preset-env", "@babel/preset-react"]
+  };
```

Comment:
- [ ] I will encourage you to set this up according to the requirement, it is meant to have the following code:
```
module.exports = {
  presets: [
    [
      '@babel/preset-env',
      {
        targets: {
          node: 'current',
        },
      },
    ],
    '@babel/preset-react',
  ],
}
```